src/Esfa.Vacancy.Register.Domain/Validation/ErrorCodes.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Esfa.Vacancy.Register.Domain.Validation
+{
+    public static class ErrorCodes
+    {
+        public static class SearchApprenticeships
+        {
+            // 30100 - 30199
```

Comment:
Unless I am not thinking things through correctly. Could we not have a dictionary so we have {Key = errorCode, value = "errorMessage"}?

Reply:
IMHO I think its over-complicating it. Have a look at the current usage:
```
RuleFor(request => request.StandardCodes)
  .NotEmpty()
  .When(request => request.FrameworkCodes == null || !request.FrameworkCodes.Any())
  .WithMessage(ErrorMessages.SearchApprenticeships.StandardAndFrameworkCodeNotProvided)
  .WithErrorCode(ErrorCodes.SearchApprenticeships.StandardAndFrameworkCodeNotProvided);
```